indent/ps1.vim

```diff
@@ -1,20 +1,87 @@
 " Vim indent file
-" Language:           Windows PowerShell
-" Maintainer:         Peter Provost <peter@provost.org>
-" Version:            2.10
-" Project Repository: https://github.com/PProvost/vim-ps1
-" Vim Script Page:    http://www.vim.org/scripts/script.php?script_id=1327"
+" Language:       Windows PowerShell
+" Maintainer:
```

Comment:
Why is @PProvost (or me, for that matter, which I should probably add) not listed? You are requested changes into this script, after all.